### DISCUSSION TITLE
Welcome to Judge-v2 Discussions!

### DISCUSSION THREAD

**Discussion by Martin-Pashov** [Martin-Pashov/Judge-v2#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by barbiekingkong** [comment]
您好，您的邮件我已经收到了，我会在第一时间给您回复的，谢谢~  鲍澎
